### PR TITLE
pause-bundle: init; kata.kata-image: optionally include pause-bundle 

### PR DESCRIPTION
**Pull request by katexochen**

### CODE REVIEW

File: packages/by-name/kata/kata-image/package.nix [hunk 1]
```diff
@@ -38,25 +40,30 @@ let
...
-
-      tar --sort=name --mtime="@$SOURCE_DATE_EPOCH" -cvf /build/rootfs-extra-tree.tar -C /build/rootfs .
-
-      mv /build/rootfs-extra-tree.tar $out
-
-      runHook postBuild
-    '';
+    buildPhase =
+      ''
+          runHook preBuild
```
Comment (burgerdev) [inline packages/by-name/kata/kata-image/package.nix:45]:
```suggestion
        runHook preBuild
```

File: packages/by-name/kata/pause-bundle/package.nix
Comment (burgerdev) [inline packages/by-name/kata/pause-bundle/package.nix:1]: Did you consider taking `nixpkgs#kubernetes.pause` as rootfs?

**Review by burgerdev — APPROVED** [review]

File: packages/by-name/kata/pause-bundle/package.nix
Comment (katexochen) [reply packages/by-name/kata/pause-bundle/package.nix:1]: Good point, I think we could do that if we aren't concerned about matching the exact version kata is using.

Comment (burgerdev) [reply packages/by-name/kata/pause-bundle/package.nix:1]: I would probably still use umoci for that (new -> copy closure of pause -> repack -> config -> unpack).

**Review by burgerdev — APPROVED** [review]
Nice!

**Pull request merged**